sentences/fr/_common.yaml

```diff
@@ -7,9 +7,49 @@ responses:
     no_device_class: "{{ area }} ne contient pas de {{ device_class }}"
     no_entity: "Aucun appareil ou entité appelé {{ entity }}"
     handle_error: "Une erreur est intervenue pendant le traitement"
-lists: {}
+lists:
+  color:
+    values:
+      - in: "blanche"
+        out: "white"
+      - in: "noire"
+        out: "black"
+      - in: "rouge"
+        out: "red"
+      - in: "orange"
+        out: "orange"
+      - in: "jaune"
+        out: "yellow"
+      - in: "verte"
+        out: "green"
+      - in: "bleue"
+        out: "blue"
+      - in: "violette"
+        out: "purple"
+      - in: "marron"
+        out: "brown"
+      - in: "rose"
+        out: "pink"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
 expansion_rules:
   name: "([le | la | les] {name} | [l']{name})"
-  area: "([le | la] {area} | [l']{area})"
+  area: "([le | la | du] {area} | [l']{area})"
+  brightness: "{brightness:brightness_pct} [pourcent]"
+  temperature: "{temperature} [degré] [{temperature_unit}]"
+  regle: "(règle | met)"
+  allume: "(allume | active | démarre)"
+  lumieres: "(la lumière | les lumières)"
+  eteint: "(éteint | éteins | désactive | stop | arrête)"
+  ventilateurs: "(le ventilateur | les ventilateurs | (le | les) ventilateurs)"
+  ventilateur_tous : "(tous les ventilateurs)"
```

Comment:
`ventilateur_tous` is not needed. We can use `[tous]` in the sentence like we do for lights.

Reply:
Fixed